package.json

```diff
@@ -28,14 +28,16 @@
     "stylelint": "bin/stylelint.js"
   },
   "files": [
-    "bin/**/*.js",
-    "lib/**/*.js",
+    "bin/**/*.{js,mjs,cjs}",
+    "lib/**/*.{js,mjs,cjs}",
     "!**/__tests__/**",
     "!lib/testUtils/**",
     "types/stylelint/index.d.ts"
   ],
   "scripts": {
     "benchmark-rule": "node scripts/benchmark-rule.mjs",
+    "build": "rollup -c",
+    "postinstall": "npm run build",
```

Comment:
TODO: Remove this `postinstall` hook when the migration finishes.

This hook is necessary to prevent `npm run lint` errors on CI.

Reply:
Won't this fire for consumers of the npm package too?